deploy/ansible/roles-sap/5.1-dbload/tasks/main.yaml

```diff
@@ -141,20 +147,10 @@
       async:                           7200
       poll:                            60
 
-    - name:                            "DBLoad Install -Wait for asynchronous job to end"
```

Comment:
Please bring this back

    - name:                            "DBLoad Install -Wait for asynchronous job to end"
      ansible.builtin.async_status:
        jid:                           '{{ dbload_results.ansible_job_id }}'
      register:                        job_result
      until:                           job_result.finished
      retries:                         120
      delay:                           60
      when:                            dbload_results.ansible_job_id is defined